crowdfunding/controllers/portal.py

```diff
@@ -0,0 +1,117 @@
+from odoo import http
+from odoo.exceptions import AccessError, MissingError
+from odoo.http import request
+from odoo.osv.expression import AND
+
+from odoo.addons.portal.controllers.portal import CustomerPortal, pager as portal_pager
+
+
+class CustomerPortal(CustomerPortal):
+    def _crowdfunding_challenge_domain(self):
+        return []
```

Comment:
```suggestion
        return ['|', ('claimed_partner_id', '=', self.env.user.partner_id.id), ('invoice_ids.partner_id', '=', self.env.user.partner_id.id)]
```

I think the way the portal view should be used is: the ones that are relevant for oneself are accessible via the portal, for everything else, go to /crowdfunding

But then I don't see this function used anywhere, would be useful to impose this in `portal_my_crowdfunding` below, right?